recipe/build.sh

```diff
@@ -12,6 +12,22 @@
 
 #Build horovod
 git submodule update --init --recursive
+
+ARCH=`uname -p`
+
+if [[ "${ARCH}" == 'x86_64' ]]; then
+    export HOROVOD_BUILD_ARCH_FLAGS='-march=nocona -mtune=haswell'
+fi
+if [[ "${ARCH}" == 'ppc64le' ]]; then
+    export HOROVOD_BUILD_ARCH_FLAGS='-mcpu=power8 -mtune=power8'
+fi
+
+if [[ $build_type == "cuda" ]]
+then
+    export HOROVOD_CUDA_HOME=$CUDA_HOME
+    export HOROVOD_CUDA_INCLUDE=/usr/include
```

Comment:
Here, I'm just adding /usr/include path in `HOROVOD_CUDA_INCLUDE` because $CUDA_HOME/include path is already being set by horovod's setup.py if it finds `HOROVOD_CUDA_HOME` set.